examples/hello_world/linux/.gitignore

```diff
@@ -0,0 +1 @@
+flutter/ephemeral
```

Comment:
My recollection is we don't check in these .gitignores, but rather push anything missing into the top level .gitignore.

Reply:
This is copied from the windows example - should it be removed there too?

Reply:
@cbracken ping